pkg/cri/config/config.go

```diff
@@ -88,6 +88,14 @@ type ContainerdConfig struct {
 	DiscardUnpackedLayers bool `toml:"discard_unpacked_layers" json:"discardUnpackedLayers"`
 }
 
+// LogConfig contains toml config related to log
+type LogConfig struct {
```

Comment:
we already have a field max_container_log_line_size.. hmm.. might want to just flatten the fields of this struct and place next to the existing log config field..

Reply:
Thanks, updated